### PR TITLE
EDU-3125: Brings into docs compliance and adds images

### PR DESCRIPTION
**Pull request by fairlydurable**
Source of truth Unni Patel

### CODE REVIEW

File: docs/production-deployment/cloud/billing-and-cost.mdx [hunk 1]
```diff
@@ -30,49 +30,92 @@ tags:
...
+Your [Billing](https://cloud.temporal.io/billing) page includes all or some of the following information, based on your plan:
 
-- [Current Balance](#current-balance): Your balance accrued thus far in the billing cycle
+- [Current balance](#current-balance): Your balance to date for this billing cycle
 - [Recent bill](#recent-bill): The amount of your most recent bill
-- [Invoice history](#invoice): A record of all past invoices
-- Credit information: If applicable, details about any credit grants, purchases, usage, and remaining credits.
+- [Invoices](#invoice): Access to all past invoices
+- [Credits](#credit-table): A record of all credit related transactions.
+  This includes details on credit grants, purchases, usage, and remaining credits, if applicable.
```
Comment (angelazhou32) [inline docs/production-deployment/cloud/billing-and-cost.mdx:45]: Unnecessary tab? We don't tab anywhere else.

File: docs/production-deployment/cloud/billing-and-cost.mdx [hunk 2]
```diff
@@ -30,49 +30,92 @@ tags:
...
 
-If you are not on a standard plan, the Billing page may not include all of the information stated above. 
+Your [Usage](https://cloud.temporal.io/usage) provides an overview of your usage across your Namespaces.
```
Comment (angelazhou32) [inline docs/production-deployment/cloud/billing-and-cost.mdx:47]: "Your Usage provides an overview of your usage across your Namespaces." reads weird to me.  Maybe something like:

The [Usage](https://cloud.temporal.io/usage) page provides an overview of your usage across your Namespaces.

File: docs/production-deployment/cloud/billing-and-cost.mdx [hunk 3]
```diff
@@ -30,49 +30,92 @@ tags:
...
+
+:::
+
+## Recent bill {#recent-bill}
+
+The "Recent Bill" card displays the previous bill amount.
+
+![Recent bill card showing a balance of $0.00](/img/billing/billing-card.png)
+
+* If you are paying invoices through the Stripe portal, you'll also see a **Pay Now** button, which takes you to the Stripe portal to complete your payment.
```
Comment (angelazhou32) [inline docs/production-deployment/cloud/billing-and-cost.mdx:68]: ? Is there a typo here?

If you are paying invoices through the Stripe portal, you're already on the Stripe portal. But you mention that if you click the Pay Now button, it takes you to the Stripe portal...which you're already on?

File: docs/production-deployment/cloud/billing-and-cost.mdx [hunk 4]
```diff
@@ -30,49 +30,92 @@ tags:
...
+
+During the current billing period, your invoice will not been finalized and the download option will not be available.
+
+:::
+
+## Credits {#credit-table}
+
+The following information appears under the credits table.
+
+- Effective At (UTC): The date when the credit grant became effective
```
Comment (angelazhou32) [inline docs/production-deployment/cloud/billing-and-cost.mdx:101]: Lines 81-87 have bullets that end with periods but lines 101-104 don't. Let's try to keep it consistent?

File: docs/production-deployment/cloud/billing-and-cost.mdx (hunk 1 above)
Comment (fairlydurable) [reply docs/production-deployment/cloud/billing-and-cost.mdx:45]: It is a continuation of the bullet so they will appear on the same line. Adding the two spaces tells Markdown that it's connected because it starts at the same indentation.

File: docs/production-deployment/cloud/billing-and-cost.mdx (hunk 2 above)
Comment (fairlydurable) [reply docs/production-deployment/cloud/billing-and-cost.mdx:47]: So I was talking on Zoom while doing this with Unni, and she wanted the two to be aligned "Your Usage" "Your Invoices", but she didn't seem to care _which_. So I can change both to "The".

File: docs/production-deployment/cloud/billing-and-cost.mdx [hunk 5]
```diff
@@ -30,49 +30,92 @@ tags:
...
+- Credit Usage: The credits used during the billing cycle.
+- Subtotal: The total amount of the invoice before any adjustments.
+- Balance Due: The amount to pay after applying credits.
 
-## Recent bill {#recent-bill}
+![Billing page showing Invoices tab](/img/billing/billing-invoices.png)
+
+:::note Current Month Invoice
+
+During the current billing period, your invoice will not been finalized and the download option will not be available.
```
Comment (angelazhou32) [inline docs/production-deployment/cloud/billing-and-cost.mdx:93]:
```suggestion
During the current billing period, your invoice will not be finalized and the download option will not be available.
```

File: docs/production-deployment/cloud/billing-and-cost.mdx (hunk 3 above)
Comment (fairlydurable) [reply docs/production-deployment/cloud/billing-and-cost.mdx:68]: Maybe rewording would help? If you pay your invoices through Stripe, you'll also see...

File: docs/production-deployment/cloud/billing-and-cost.mdx (hunk 1 above)
Comment (angelazhou32) [reply docs/production-deployment/cloud/billing-and-cost.mdx:45]: Thanks for clarifying!

File: docs/production-deployment/cloud/billing-and-cost.mdx (hunk 4 above)
Comment (fairlydurable) [reply docs/production-deployment/cloud/billing-and-cost.mdx:101]: Argh! My nemesis. None of the bullet lines should have punctuation. I'll audit the file for all bullets that I accidentally introduced

Comment (fairlydurable) [reply docs/production-deployment/cloud/billing-and-cost.mdx:101]: Audited and done. Thank goodness for regex.

File: docs/production-deployment/cloud/billing-and-cost.mdx (hunk 5 above)
Comment (fairlydurable) [reply docs/production-deployment/cloud/billing-and-cost.mdx:93]: Fixed.

File: docs/production-deployment/cloud/billing-and-cost.mdx (hunk 3 above)
Comment (fairlydurable) [reply docs/production-deployment/cloud/billing-and-cost.mdx:68]: Fixed

File: docs/production-deployment/cloud/billing-and-cost.mdx (hunk 2 above)
Comment (fairlydurable) [reply docs/production-deployment/cloud/billing-and-cost.mdx:47]: Aligned both spots to "The". Thanks.

**Review by angelazhou32 — APPROVED** [review]
